Add bonsai-operators docs section (#32)
ucl-open/rigs: docs/bonsai-operators/UclOpen.Devices/harp-timestamp-generator-gen3.md

| `port_name` | `str` | - | Serial port the device is connected to (e.g. `COM3`) |
| `timer_frequency` | `TimerFrequency` | `Timer1000Hz` | Rate at which timestamp events are sent to Bonsai |

> `TimerFrequency` is used to set the rate at which clock events are generated on the Harp device: `Disabled`, `Timer50Hz`, `Timer100Hz`, `Timer200Hz`, `Timer500Hz`, `Timer1000Hz`. These are collected by Bonsai and published to a subject that can then be used as the timestamps for the rest of your workflow. 

Reviewer: ```suggestion
`TimerFrequency` is used to set the rate at which clock events are generated on the Harp device: `Disabled`, `Timer50Hz`, `Timer100Hz`, `Timer200Hz`, `Timer500Hz`, `Timer1000Hz`. These are exposed in Bonsai by a subject that can then be used as the timestamps for the rest of your workflow. 
```

## Using the timebase in other modules

To stamp data from a non-Harp device with the hardware clock, subscribe to the `Timestamp` subject from the `Timestamps` group, pair it with your data stream using `WithLatestFrom`, then pass the result to `CreateTimestamped`:

> > **Note:** Unlike native Harp devices, this timestamp is assigned in software when the event arrives on the PC, not in hardware at the moment of acquisition. It is therefore subject to PC scheduling jitter and USB latency. For applications requiring sub-millisecond timing accuracy, use a Harp device directly.

Reviewer: This is a very useful general point about timestamping, I wonder if it should be in a more general section in future as it is not specific to the TimestampGenerator hardware.

Author: Yes, it could be repeated in the Timestamps.bonsai sub workflow for the behaviour board too. A general section describing timestamps and synching in general? For a future PR?

Reviewer: Doesn't matter for this PR - I'll add an issue as a reminder for future.

Reviewer: https://github.com/ucl-open/ucl-open/issues/34
